internal/docs/openapi.go

```diff
@@ -0,0 +1,127 @@
+package docs
```

Comment:
Is it idiomatic to put swagger stuff inside a `docs` package in go?
Because currently there's development.md documentation file inside the docs/ directory.
I don't want to cram both go code and MD files inside the docs/ folder so either the go code can be moved to a different directory or the MD file can be.

Reply:
I love the format of this project, where you keep your markdown file outside to find easily 
all of your api docs / go docs can go to dir/docs , thats my suggestion, open for your thoughts.

https://github.com/agentgateway/agentgateway

Reply:
also this is in internal/docs not docs in root of repo